ffbs-parker-nodeconfig/Makefile

```diff
@@ -21,19 +21,5 @@ define Package/ffbs-parker-nodeconfig/description
   It keeps track of the node's connectivity and changes the state as needed.
 endef
 
-define Build/Prepare
-	mkdir -p $(PKG_BUILD_DIR)
-endef
-
-define Build/Configure
-endef
-
-define Build/Compile
-endef
-
-define Package/ffbs-parker-nodeconfig/install
-	$(CP) ./files/* $(1)/
-endef
-
-$(eval $(call BuildPackage,ffbs-parker-nodeconfig))
+$(eval $(call BuildPackageGluon,ffbs-parker-nodeconfig))
```

Comment:
Unrelated change to VPN provider? Please split commit or comment.

Reply:
This makes sure that the check_site.lua is actually executed on build.
See also: https://gluon.readthedocs.io/en/latest/dev/packages.html#provided-macros

Reply:
With your explanation it makes sense. Thx.